### DISCUSSION TITLE
fp16 and int8 support for vulkan backend

### DISCUSSION THREAD

**Discussion by vjayd** [Tencent/ncnn#4785] [Ideas]
Do we plan to make the quantized model int8 run on the Vulkan backend? Especially generative AI models are very slow and need extremely fast technique to run on Vulkan device

**Comment by nihui** [comment]
Yes, but will wait for upstream glslang to support int8 dot product extension
Without this extension, int8 would be very slow

**Reply by Mek101** [reply]
You mean https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_shader_integer_dot_product.html ?

**Comment by nihui** [comment]
int8 support for vulkan https://github.com/Tencent/ncnn/pull/6751
